components/admin/requests/forms/new-applications/GuardianInformationForm.tsx

```diff
@@ -0,0 +1,203 @@
+              onChange={event =>
+                onChange({
+                  ...guardianInformation,
+                  guardianRelationship: event.target.value,
+                })
+              }
+            />
+          </FormControl>
+        </Stack>
+      </Box>
+
+      <Divider borderColor="border.secondary" />
+
+      {/* Address Section */}
+
+      <Box paddingTop="32px">
+        <Text as="h3" textStyle="heading" paddingBottom="24px">
+          {'Address '}
+        </Text>
+
+        <FormControl isRequired paddingBottom="24px">
+          <FormLabel>{'Address line 1'}</FormLabel>
+          <Input
+            value={guardianInformation.addressLine1}
+            onChange={event =>
+              onChange({
+                ...guardianInformation,
+                addressLine1: event.target.value,
+              })
+            }
+          />
+          <FormHelperText color="text.seconday">
+            {'Street Address, P.O. Box, Company Name, c/o'}
+          </FormHelperText>
+        </FormControl>
+
+        <FormControl paddingBottom="24px">
+          <FormLabel>
+            {'Address line 2 '}
+            <Box as="span" textStyle="caption" fontSize="sm">
+              {'(optional)'}
+            </Box>
+          </FormLabel>
+          <Input
+            value={guardianInformation.addressLine2 || ''}
+            onChange={event =>
+              onChange({
+                ...guardianInformation,
+                addressLine2: event.target.value,
+              })
+            }
+          />
+          <FormHelperText color="text.seconday">
+            {'Apartment, suite, unit, building, floor, etc'}
+          </FormHelperText>
+        </FormControl>
+
+        <Stack direction="row" spacing="20px">
+          <FormControl isRequired>
+            <FormLabel>{'City'}</FormLabel>
+            <Input
+              value={guardianInformation.city}
+              onChange={event =>
+                onChange({
+                  ...guardianInformation,
+                  city: event.target.value,
+                })
+              }
+            />
+          </FormControl>
+
+          <FormControl isRequired>
+            <FormLabel>{'Postal code'}</FormLabel>
+            <Input
+              value={guardianInformation.postalCode}
+              onChange={event =>
+                onChange({
+                  ...guardianInformation,
+                  postalCode: event.target.value,
+                })
+              }
+            />
+            <FormHelperText color="text.seconday">{'Example: X0X 0X0'} </FormHelperText>
+          </FormControl>
+        </Stack>
+
+        <Text as="h3" textStyle="heading" paddingBottom="20px">
+          {'Upload POA File'}
+        </Text>
+        <Text color="text.seconday">
+          {'Only ONE file can be added. Files must be .pdf and can be a maximum of 5MB in size.'}{' '}
+        </Text>
```

Comment:
Can we add a TODO comment to implement the file upload functionality and file list?